### DISCUSSION TITLE
query_string_filter not returning objects without name

### DISCUSSION THREAD

**Discussion by quargh** [komoot/photon#568] [Ideas]
Hello,

I have noticed that the reverse option is returning only poor results when querying objects like query_string_filter=osm_value:parking, telephone and so on, because apparently objects won't be listed if they do not have a name tag. Since objects like parking, telephone and many others often do not have name tags, wouldn't it be a good idea to include all objects in the results, regardless if they have a name or not.

**Comment by lonvia** [comment]
These objects are not in the database in the first place because Photon is first and former a search engine. If you are looking for OSM objects with a certain tag around a given location, have a look at [Overpass API](https://overpass-api.de/).
